include/gz/rendering/Utils.hh

```diff
@@ -111,6 +112,10 @@ namespace gz
     gz::math::Matrix3d projectionToCameraIntrinsic(
         const gz::math::Matrix4d &_projectionMatrix,
         double _width, double _height);
+    
+    // \brief convert a rgb image data into bayer image data
+    GZ_RENDERING_VISIBLE
+    void ConvertRGBToBayer(Image &_image);
```

Comment:
for global functions, we use a lowercase first letter for the function name. Can you also make this return a copy of the image instead of modifying the source image, i.e. `Image convertRGBToBayer(const Image &_image);`